todoapp/projects/models.py

```
@@ -24,4 +43,16 @@ class ProjectMember(models.Model):
     Add string representation for this model with project name and user email/first name.
     """
 
+    project = models.ForeignKey(Project, on_delete=models.CASCADE)
+    member = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
+
+    def __str__(self):
```

Ruff (0.12.2)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)